cli/config/config.go

```diff
@@ -117,15 +119,73 @@ func InitViperConfig() {
 		_, err := AppFs.Stat(Path)
 		if err == nil {
 			log.Debug().Str("configfile", viper.ConfigFileUsed()).Msg("try to load local config file")
-			if err := viper.ReadInConfig(); err == nil {
-				LoadedConfig = true
+
+			// Check if this is a WIF config file
+			isWif, wifErr := IsWifConfigFormat(Path)
```

Comment:
is it not possible to make wif config a subset of our normal config file?

Reply:
Maybe, I will look into that a bit.